libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/impl/AWSEmailProvider.java

```diff
@@ -27,77 +27,20 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.mail.Session;
-import javax.mail.internet.InternetAddress;
-import javax.mail.internet.MimeBodyPart;
 import javax.mail.internet.MimeMessage;
-import javax.mail.internet.MimeMultipart;
 import java.io.ByteArrayOutputStream;
 import java.nio.ByteBuffer;
 import java.util.Collection;
-import java.util.Properties;
-
-import static com.yahoo.athenz.common.server.notification.NotificationServiceConstants.CHARSET_UTF_8;
-import static com.yahoo.athenz.common.server.notification.NotificationServiceConstants.HTML_LOGO_CID_PLACEHOLDER;
 
 public class AWSEmailProvider implements EmailProvider {
     private static final Logger LOGGER = LoggerFactory.getLogger(AWSEmailProvider.class);
     private final AmazonSimpleEmailService ses;
 
     @Override
-    public boolean sendEmail(String subject, String body, boolean status, Collection<String> recipients, String from, byte[] logoImage) {
+    public boolean sendEmail(boolean status, Collection<String> recipients, String from, MimeMessage mimeMessage) {
```

Comment:
As @abvaidya suggested, extracted the MimeMessage construction so providers will need need to implement it.